### PR TITLE
UI Fixes

### PR DESCRIPTION
**Pull request by Suvink**
Redesigned UI components

### CODE REVIEW

File: src/routes/Home/components/HomePage/HomePage.js [hunk 1]
```diff
@@ -1,50 +1,150 @@
...
-              <li>
-                They will come and fulfill your needs
-              </li>
-            </ol>
-          </Grid>
+    <div>
+      <Grid container spacing={3} className={classes.mainContainer}>
+        <Grid item xs={12} sm={6} className={classes.leftSide}>
+          <Image
+            src="https://media1.giphy.com/media/LpW8NlEe3KBlXoLpXd/source.gif"
```
Comment (agentmilindu) [inline src/routes/Home/components/HomePage/HomePage.js:25]: Let's add this image to our firebase static files and refer it form there

File: src/routes/Home/components/HomePage/HomePage.js [hunk 2]
```diff
@@ -1,50 +1,150 @@
...
+            You can Log In and start adding your basic needs in this distressing
+            times.
+          </Typography>
+
+          <Card className={classes.cardRoot}>
+            <CardMedia
+              className={classes.cover}
+              component="img"
+              width="50%"
+              image="https://image.flaticon.com/icons/svg/701/701997.svg"
```
Comment (agentmilindu) [inline src/routes/Home/components/HomePage/HomePage.js:76]: Let's add this image to our firebase static files and refer it from there

File: src/routes/Account/components/AccountEditor/AccountEditor.js [hunk 3]
```diff
@@ -37,11 +39,25 @@ function AccountEditor() {
...
+          <Typography variant="h6" color="secondary">
+            Linked Accounts
+          </Typography>
+          <div className={classes.linkedItem}>
+            <PermIdentitySharpIcon color="secondary" />
+            <Typography
+              variant="h6"
+              color="secondary"
+              className={classes.userName}>
+              Test User
```
Comment (agentmilindu) [inline src/routes/Account/components/AccountEditor/AccountEditor.js:57]: This should show the linked set of Auth providers here, Try if you can figure it out where it got break. I'll help.

**Comment by agentmilindu** [comment]
Superb Job! Nicely done!

File: src/routes/Account/components/AccountForm/AccountForm.js [hunk 4]
```diff
@@ -27,46 +27,59 @@ function AccountForm({ account, onSubmit }) {
...
       )}
-      <Button
-        color="primary"
-        type="submit"
-        variant="contained"
-        disabled={isSubmitting || !isValid}>
-        {isSubmitting ? 'Saving' : 'Save'}
-      </Button>
+
+      <div style={{ textAlign: 'left!important' }}>
```
Comment (agentmilindu) [inline src/routes/Account/components/AccountForm/AccountForm.js:73]: Let's move this style to a class and refer here

Comment (Suvink) [reply src/routes/Account/components/AccountForm/AccountForm.js:73]: Done

File: src/routes/Home/components/HomePage/HomePage.js (hunk 1 above)
Comment (Suvink) [reply src/routes/Home/components/HomePage/HomePage.js:25]: Done

File: src/routes/Home/components/HomePage/HomePage.js (hunk 2 above)
Comment (Suvink) [reply src/routes/Home/components/HomePage/HomePage.js:76]: Done

**Pull request merged**